spec/mailers/bulk_url_upload_results_mailer_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'a bulk upload notification email' do
+  it 'has the correct subject' do
+    expect(mail.subject).to eq("Bulk URL upload results for #{filename}")
+  end
+
+  it 'has the correct recepient' do
+    expect(mail.to).to eq([user.email])
+  end
+
+  it 'has the correct from header' do
+    expect(mail.from).to eq([ApplicationMailer::DELIVER_FROM_EMAIL_ADDRESS])
+  end
+
+  it 'has the correct reply-to' do
+    expect(mail.reply_to).to eq([ApplicationMailer::REPLY_TO_EMAIL_ADDRESS])
+  end
+end
+
+RSpec.describe BulkUrlUploadResultsMailer, type: :mailer do
```

Comment:
No need to change now, but you can omit the `type: :mailer`, as we use RSpec's `infer_spec_type_from_file_location!` configuration.
https://relishapp.com/rspec/rspec-rails/docs/directory-structure

Reply:
OK. That was generated code.

Reply:
👍